util/EIP-1024/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
There is already a copy in the main directory.

Reply:
ok